docs/api.rst

```diff
@@ -144,6 +145,105 @@ Variable recombination rates
 .. autoclass:: msprime.RecombinationMap
     :members:
 
+.. _sec_api_simulation_models:
+
+*****************
+Simulation models
+*****************
+
+The default simulation model in ``msprime`` is the standard coalescent with recombination
+model. We also support a number of different models, which are documented in this section.
+
+Simulations models are specified using the ``model`` parameter to
+:func:`.simulate`. This parameter can either take the form of a
+string describing the model (e.g. ``model="smc"``) or an instance of a
+model definition class (e.g ``model=msprime.SmcApproxCoalescent(1)``).
+The available models are documented in the following subsections.
+
+A key element of simulation models in ``msprime`` is the concept
+of a "reference population size". Simulation models such as
+the coalescent are defined
+in terms of "scaled time". Time in the standard diploid coalescent
+is measured in units of :math:`4 N_e` generations, and
```

Comment:
Isn't time in units of `N_e` generations? This phrase seems to contradict [these docs](https://msprime.readthedocs.io/en/stable/api.html?highlight=diploid#simulation-model).

Reply:
I think this point - that under the hood, many models just treat Ne as a time scaling - is a side note, and is kinda confusing to put right up the top of this. All the simulation models have a *population size*, and they all mean the same thing; and maybe by the way in some models this is just a time scaling.

Reply:
Good points, thanks. I'll clear this up.